trompace/client/client_itemlist.py

```diff
@@ -378,15 +378,18 @@ def insert_listitem_itemlist(contributor: str, name: str, description: str,
     itemlist_obj = itemlist_obj[0]
     itemlist_id = itemlist_obj["identifier"]
     itemlist_elements = itemlist_obj["itemListElement"]
+    itemlist_elements = sorted(itemlist_elements, key=lambda k: k['position'])
```

Comment:
we should add a comment here which explains why we do this. Something like:

`# We can't guarantee the order that items come out of the CE, so explicitly sort them`

A question - what happens if the position field is null?

Reply:
It fails. However, when inserting a ListItem in a existing ItemList, we could assume that every ItemList already has a position set. Anyway, I can add a check and raising an Error if a ListItem with position null is present.